testing/trino-product-tests-launcher/src/main/java/io/trino/tests/product/launcher/env/environment/EnvMultinodeHiveCaching.java

```diff
@@ -64,7 +66,11 @@ public EnvMultinodeHiveCaching(
         super(ImmutableList.of(standard, hadoop));
         this.dockerFiles = requireNonNull(dockerFiles, "dockerFiles is null");
         this.configDir = dockerFiles.getDockerFilesHostDirectory("conf/environment");
-        this.imagesVersion = requireNonNull(environmentConfig, "environmentConfig is null").getImagesVersion();
+        this.baseTrinoImage = format(
```

Comment:
i don't understand yet why we would want baseTrinoImage to be configurable.
seems not needed, so would prefer to revert all changes related to that

Reply:
reverted